flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/WebMonitorEndpoint.java

```diff
@@ -675,26 +704,6 @@ public WebMonitorEndpoint(
 		return handlers;
 	}
 
-	@Nonnull
-	private ChannelInboundHandler createStaticFileHandler(
-			Time timeout,
-			File fileToServe) {
-
-		if (fileToServe == null) {
-			return new ConstantTextHandler("(file unavailable)");
```

Comment:
@vthinkxie could you see it?

Reply:
https://github.com/apache/flink/blob/d39f5b33221c813e6b9478001675cb56cf361236/flink-runtime-web/web-dashboard/src/app/app.interceptor.ts#L38
it could work

Reply:
If it works, we could delete ConstantTextHandler.